systemtests/utils/vagrantcommand.go

```diff
@@ -45,6 +35,11 @@ func (c *VagrantCommand) getCmd(cmd string, args ...string) *exec.Cmd {
 	return osCmd
 }
 
+type VagrantCommand struct {
```

Comment:
looks like the structure definition got moved somewhere in between it's member functions, suggest to move it to top.